### DISCUSSION TITLE
Where should home be?

### DISCUSSION THREAD

**Discussion by cristian-rivera** [Jyers/Marlin#966] [Q&A] [answered]
Given the following:
- V1.3.3 BLTouch 5x5 4.2.2
- Probe offset of -44/-7
- Home offset of -5/-10 ([MicroSwiss Direct Drive](https://store.micro-swiss.com/products/micro-swiss-direct-drive-extruder))
- Bed size 230/230 (Jyers default, actual 235/235)

Using the LCD, if I "Home All Axis's", where should my position end up?

Expected?

230 / 2 = 115 + 44 (Probe offset) + 5 (DD Home offset) = 164

230 / 2 = 115 + 7 (Probe offset) + 10 (DD Home offset) = 132

Actual according to LCD
159/127

**Comment by Jyers** [comment, marked as answer]
This is how Marlin sets up z safe homing
![image](https://user-images.githubusercontent.com/76993396/120903535-a1bb8a00-c614-11eb-9462-f535485f6645.png)

Don't ask me why, but here it is 😅

**Reply by cristian-rivera** [reply]
Interesting, what do you recommend here?

Should I keep my bed size at 230 or go back to the stock 235? Out of curiosity, why is this set to 230 in this firmware?

Should I keep my home offsets at -5/-10 as recommended by MicroSwiss when installing their DirectDrive extruder?

Ideally, I just want a centered print in CURA to be at the center of my bed physically and for all mesh creating to be valid even with my upgraded DirectDrive, thanks!

**Reply by Jyers** [reply]
No matter what, as long as your probe offsets are correct the mesh should be fine. If you want your print to be totally centered you have a few options. Here's what I did:

![image](https://user-images.githubusercontent.com/76993396/120903920-0677e400-c617-11eb-97dc-e6094185d6c7.png)

This lets you account for where the nozzle is when it is at the endstops (mine is at pos x = 4, y = 7) while still getting your maximum possible print area.

Besides that you can also set your bed size to 235 and make sure your home offsets are exact. The bed size is 230 by default as most people don't bother to set their home offsets and the nozzle on these printers tend to be a bit into the bed at the endstops.

**Reply by Reggi3** [reply]
That - 10 ) /2 bit has always bothered me... that - 10 doesn't appear to be a default setting, it's just there but no one seems to know why it's there.

**Reply by cristian-rivera** [reply]
@Jyers thank you, I have forked the repo and compiled my own version!

**Reply by cristian-rivera** [reply]
> That - 10 ) /2 bit has always bothered me... that - 10 doesn't appear to be a default setting, it's just there but no one seems to know why it's there.

It does seem a bit odd
